pagopa/io-dev-api-server
# fix: SPID logo retrieval

## Short description
This PR adds missing SPID provider logos and fixes the retrieval them. 

## How to test
Uninstall App IO if previously installed. Try a SPID login with App IO. In the SPID IdP selection screen you should see all providers logo.

<details><summary>Demo</summary>
<p>

<img width="250" src="https://github.com/pagopa/io-dev-api-server/assets/16268789/91482a02-7c05-4d5e-9da5-f117161e1b12" />

</p>
</details>
